app/src/main/java/com/alexis/myanimecompanion/ui/search/SearchListAdapter.kt

```diff
@@ -46,4 +46,4 @@ class SearchListAdapter(private val clickListener: ClickListener) :
         }
     }
 
-}
```

Comment:
Revert this, please. You shouldn't be modifying files where you haven't changed any code